websites/L/Learnjavascript/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "293803241110503426",
+		"name": "weemizo"
+	},
+	"contributors": [
+		{
+			"name": "weemizo",
```

Comment:
You dont need to add yourself as contributor if you are the author

Reply:
Fixed